editor/modes/visual-editor/block.js

```diff
@@ -58,13 +57,13 @@ function VisualEditorBlock( props ) {
 			tabIndex="0"
 			onFocus={ onSelect }
 			onBlur={ maybeDeselect }
-			onKeyDown={ onDeselect }
-			onMouseEnter={ onMouseEnter }
+			onKeyDown={ onStartTyping }
+			onMouseMove={ onMouseMove }
```

Comment:
`onMouseMove` is an incredibly noisy event (watch Redux DevTools while moving). Is the idea here that we want the block to regain its hover effects when moving the mouse while typing? Maybe we can at least add a condition to only trigger the callback if `isTyping` is true?